diagnostic.md

```diff
@@ -32,23 +37,32 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
+Getter methods:
+  $('#content').text()
+  $('#content').length()
+Setter methods:
+  $('#content').text('change content text')
+  )
+  $('#content').length(10)
 <!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+The DOM is a current representation of a page.
```

Comment:
Yes and it is arranged in a tree-like structure so that every node is either a child, parent, or both to other nodes.